tests/test_client.py

```diff
@@ -461,6 +461,20 @@ def test_user_no_data_returns_empty(client, sample_org_data, api_user3: KhojApiU
     assert response.json() == []
 
 
+@pytest.mark.django_db(transaction=True)
+def test_chat_with_unauthenticated_user(chat_client_with_auth, api_user2: KhojApiUser):
+    # Arrange
+    headers = {"Authorization": f"Bearer {api_user2.token}"}
```

Comment:
This seems to be failing with an internal server 500 error

Reply:
Yeah, investigating that. Not sure why it's failing in the workflow but not locally on my machine

Reply:
Never mind, the issue was trivial. We weren't checking for required API key environment variable before trying to run the test. It's resolves now